SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/SpatialLatencyTracer.h"
+
+#include "Async/Async.h"
+#include "Engine/World.h"
+#include "EngineClasses/SpatialGameInstance.h"
+#include "Interop/Connection/OutgoingMessages.h"
+#include "Utils/SchemaUtils.h"
+
+#include <sstream>
+
+DEFINE_LOG_CATEGORY(LogSpatialLatencyTracing);
+
+// Stream for piping trace lib output to UE output
+class UEStream : public std::stringbuf
+{
+	int sync()
+	{
+		UE_LOG(LogSpatialLatencyTracing, Verbose, TEXT("%s"), *FString(str().c_str()));
```

Comment:
For my own understanding - this is just meant to take the input to some ostream object and transfer it all to the Unreal logger then?

Reply:
Yep